Source/WebCore/page/Page.cpp

```diff
@@ -549,7 +549,7 @@ Ref<DOMRectList> Page::nonFastScrollableRectsForTesting()
     return DOMRectList::create(quads);
 }
 
-Ref<DOMRectList> Page::touchEventRectsForEventForTesting(const String& eventName)
+Ref<DOMRectList> Page::touchEventRectsForEventForTesting(EventTrackingRegions::Event eventName)
```

Comment:
Let's also change the variable name from `eventName` to `event`.

Reply:
Changed.